CMakeLists.txt

```diff
@@ -48,3 +48,5 @@ set(HEADERS
 install(DIRECTORY include/ DESTINATION /usr/local/include/AppwriteSDK)
 install(FILES ${HEADERS} DESTINATION /usr/local/include/AppwriteSDK)
 install(TARGETS AppwriteSDK ARCHIVE DESTINATION /usr/local/lib)
+
+
```

Comment:
I can still see line additions. Please donot touch any file unless u do some changes in it.